Update the ReadMe, frame Documentation, and License info of the repo (#32)
robotology/walking-teleoperation: README.md

# walking-teleoperation

> The whole-body teleoperation architecture as shown as following.

Reviewer: ```suggestion
The whole-body teleoperation architecture is shown in the following.
```

> The human user receives visual feedback from the robot environment by streaming the robot camera images through the \emph{Oculus Headset}.

Reviewer 2: `emph` should not work. `__Oculus Headset__` should be fine

- ``iCubGenova04``
- ``iCubGenova02``

## :eyeglasses: Reference paper

> You can read more about the work in [this linke.](https://arxiv.org/pdf/1909.10080.pdf)

Reviewer: ```suggestion
You can read more about the work at [this link.](https://arxiv.org/pdf/1909.10080.pdf)
```

If you're going to use this package for your work, please quote it within any resulting publication:
```
@inproceedings{Whole-Body-2019,
author = {Darvish, Kourosh and Tirupachuri, Yeshasvi and Romualdi, Giulio and Rapetti, Lorenzo and Ferigo, Diego and Chavez, Francisco Javier Andrade and Pucci, Daniele},
 title = {Whole-Body Geometric Retargeting for Humanoid Robots},
booktitle = {Proceedings of the 2019 IEEE/RAS International Conference on Humanoid Robots (Humanoids)},
year = {2019},
month={October},
address = {Toronto, Canada},
}
```
## :dollar: License

> Walking-Teleoperation is licensed under either the GNU Lesser General Public License v3.0 :

Reviewer: ```suggestion
walking-teleoperation is licensed under either the GNU Lesser General Public License v3.0 :
```
